### DISCUSSION TITLE
Setup with Discovery server

### DISCUSSION THREAD

**Discussion by ValkyrX** [turtlebot/turtlebot4#224] [Q&A]
Hi, because my corporate wifi is crappy i am setting up a discovery server setup, the pi4 and the create3 communicate fine, but when i follow the tutorial and i try and run the script (humble on ubutnu 22.04) i get a load of errors and sed cant write etc, doe sthe script not work any more and if not what do i do to make the remote computer (not pi4 ) be able to see the topics ? i have installed ros2 humble and installed the turtlebot packages but it simply wont run the script to configure for discovery server

Help :-)

Thanks

**Comment by roni-kreinin** [comment]
@ValkyrX Are you running Ubuntu 22.04 natively or on a VM? Also could you share the errors that you are seeing?

**Comment by bmaxdk** [comment]
Did you configure the ros2 node to use the discovery server? On both the server and clients, you will need to set specific environment variables to configure the DDS middleware.
